MM2RandoLib/Resources/Asm/Options/merciless.asm

```diff
@@ -0,0 +1,75 @@
+.include "mm2r.inc"
+
+.segment "BANKB"
+
+.org $a564
+	nop ; Was lda 02 E5 74
+	nop ; Was bne $a59c
+	nop
+	nop
+
+.segment "BANKE"
+
+.org $8a5a
+	nop ; Was ldy IframesLeft
+	nop ; Was bne $8a65
+	nop
+	nop
+
+.org $8cbe
+	nop ; Was ldy IframesLeft
+	nop ; Was bne $8cc9
+	nop
+	nop
+
+.org $9e4b
+	jmp CheckLaserContact ; Was lda IframesLeft
+	nop ; Was bne $9e80
+
+CheckLaserContactRet:
+
+.reloc
+CheckLaserContact:
+	; Quick lasers do weird things with iframes
```

Comment:
Can you explain what this means in the comment?

Reply:
Umm. I don't remember the details, but I recall that I had to do Quick lasers differently than other cases, because there was some special behavior around iframe counts of 0 and 1.